src/metabase/premium_features/task/send_metering.clj

```diff
@@ -0,0 +1,35 @@
+(ns metabase.premium-features.task.send-metering
+  "Scheduled task that sends metering events for billing purposes."
+  (:require
+   [clojurewerkz.quartzite.jobs :as jobs]
+   [clojurewerkz.quartzite.schedule.simple :as simple]
+   [clojurewerkz.quartzite.triggers :as triggers]
+   [metabase.premium-features.token-check :as token-check]
+   [metabase.task.core :as task]
+   [metabase.util :as u]
+   [metabase.util.log :as log]))
+
+(set! *warn-on-reflection* true)
+
+(task/defjob ^{:doc "Send metering events for billing purposes"}
+  SendMeteringEvents
+  [_]
+  (log/debug "Running metering events task")
+  (token-check/send-metering-events!))
+
+(def ^:private job-key "metabase.task.send-metering.job")
+(def ^:private trigger-key "metabase.task.send-metering.trigger")
+
+(defmethod task/init! ::SendMeteringEvents
```

Comment:
I've moved the metering events to a tasks that runs every 12 hours instead of being tied to the token check, so we could run it more frequently if desired.
 
I'm raising this because I need feedback on whether this is desirable or not, and whether instead we still want the metering submission to be tied to token checks (and crucially, fail token check if metering submission fails)